tests/transactions/tests.py

```diff
@@ -443,8 +443,25 @@ def test_atomic_does_not_leak_savepoints_on_failure(self):
                 # This is expected to fail because the savepoint no longer exists.
                 connection.savepoint_rollback(sid)
 
-    @skipIf(connection.features.autocommits_when_autocommit_is_off,
-            "This test requires a non-autocommit mode that doesn't autocommit.")
+
+@skipIf(
+    connection.features.autocommits_when_autocommit_is_off,
+    "This test requires a non-autocommit mode that doesn't autocommit."
+)
+class NonAutocommitTests(TransactionTestCase):
+
+    available_apps = []
+
+    def test_orm_query_after_error(self):
```

Comment:
As far as I know, non-automcommit mode hasn't been usable since 1.6. The docs say something that boils down "you're on your own and you aren't going to have fun" because no one could offer a use case for running a web app without autocommit so I didn't put effort into supporting that.

If there's an additional regression from 1.9 to 1.10 (unclear to me), the fix should be backported.

If not, I don't think it should be backported.